galite-core/src/main/kotlin/org/kopi/galite/form/VBlockDefaultOuterJoin.kt

```diff
@@ -77,22 +88,13 @@ class VBlockDefaultOuterJoin(block: VBlock) {
                   }
                 } else {
                   if (rootTable == table) {
-                    val mainTable = object : Table(tables!![table].tableName) {
-                    }
-                    val joinTable = object : Table(tables!![field.getColumn(j)!!.getTable()].tableName) {
-                    }
+                    val joinTable = tables!![field.getColumn(j)!!.getTable()]
+
                     // start of an outer join
-                    addToJoinedTables(field.getColumn(j)!!.column!!.table)
-
-                    transaction {
-                      joinBuffer!!.also {
-                        it.add(Join(mainTable,
-                                    joinTable,
-                                    JoinType.LEFT,
-                                    field.getColumn(tableColumn)!!.column!! eq field.getColumn(j)!!.column!!)
-                        )
-                      }
-                    }
+                    addToJoinedTables(field.getColumn(j)!!.getTable_())
+                    joinTables.join(joinTable, JoinType.INNER, field.getColumn(tableColumn)!!.column,
```

Comment:
I am not sure about the jointype, in kopi it is a left outer join, please check it